WMFComponents/Sources/WMFComponents/Components/Profile/ProfileViewController.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+import UIKit
+
+class ProfileViewController: UIViewController {
```

Comment:
Similar note here - name this "WMFProfileViewController" since it lives in WMFComponents.